fdbserver/QuietDatabase.actor.cpp

```diff
@@ -195,6 +231,45 @@ ACTOR Future<vector<StorageServerInterface>> getStorageServers( Database cx, boo
 	}
 }
 
+ACTOR Future<vector<WorkerInterface>> getStorageWorkers( Database cx, Reference<AsyncVar<ServerDBInfo>> dbInfo, bool localOnly ) {
+	Future<std::vector<StorageServerInterface>> serversFuture = getStorageServers(cx);
```

Comment:
As a weird minor optimization, if you make all the state variables first, the actor compiler will directly initialize them rather than default initialize and then copy.